### ISSUE TITLE
dev-python/jupyter-jsmol diseappeared from repo

### ISSUE THREAD

**Issue by CaptainBloodz** [cschwan/sage-on-gentoo#740]
eix dev-python/jupyter-jsmol
[?] dev-python/jupyter-jsmol [1]
     Available versions:  ~2022.1.0^t {test PYTHON_TARGETS="python3_10 python3_11"}
     Installed versions:  2022.1.0^t(16:29:30 02/05/2023)(-test PYTHON_TARGETS="python3_10 -python3_11")

I guess it's just temporary, or is it?

**Comment by kiwifb** [comment]
```
$ eix jupyter-jsmol
[I] dev-python/jupyter-jsmol [1]
     Available versions:  (~)2022.1.0 {test}
     Installed versions:  2022.1.0^t(13:19:37 05/03/23)(-test PYTHON_TARGETS="python3_11 -python3_10")
     Homepage:            https://github.com/fekad/jupyter-jsmol
     Description:         jupyter jsmol widget viewer

[1] "sage-on-gentoo" /var/db/repos/sage-on-gentoo
```
here. We have just changed from `jupyter_jsmol` to jupyter-jsmol` and it has been done properly as far as I can see. Did you get any migration message when you synced? What do you use to sync the tree?

**Comment by n1ghtbyte** [comment]
After running `eix-update` as root, you'll be able to get the same output as @kiwifb.
You can also update the eix cache on each sync you do.

**Comment by CaptainBloodz** [comment]
Oops my bad...
package.use and package.accept_keyword according lines require jupyter_jsmol to jupyter-jsmol replacement.
Once done  eix outputs is now: 

eix dev-python/jupyter-jsmol
[I] dev-python/jupyter-jsmol [1]
     Available versions:  (~)2022.1.0^t {test PYTHON_TARGETS="python3_10 python3_11"}
     Installed versions:  2022.1.0^t(16:29:30 02/05/2023)(-test PYTHON_TARGETS="python3_10 -python3_11")

with heading question mark gone. Nice
